app/src/main/java/cm/aptoide/pt/home/BottomNavigationNavigator.java

```diff
@@ -0,0 +1,153 @@
+package cm.aptoide.pt.home;
+
+import android.support.v4.app.Fragment;
+import cm.aptoide.pt.dataprovider.model.v7.Event;
+import cm.aptoide.pt.dataprovider.ws.v7.store.StoreContext;
+import cm.aptoide.pt.home.apps.AppsFragment;
+import cm.aptoide.pt.navigator.FragmentNavigator;
+import cm.aptoide.pt.search.analytics.SearchAnalytics;
+import cm.aptoide.pt.search.analytics.SearchSource;
+import cm.aptoide.pt.search.view.SearchResultFragment;
+import cm.aptoide.pt.store.view.my.MyStoresFragment;
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * Created by D01 on 02/04/18.
+ */
+
+public class BottomNavigationNavigator {
+
+  private final static String EVENT_ACTION =
+      "https://ws75.aptoide.com/api/7/getStoreWidgets/store_id=15/context=stores";
+  private final FragmentNavigator fragmentNavigator;
+  private final String defaultStoreName;
+  private final BottomNavigationAnalytics bottomNavigationAnalytics;
+  private final SearchAnalytics searchAnalytics;
+  private ArrayList<Integer> bottomNavigationItens;
+
+  public BottomNavigationNavigator(FragmentNavigator fragmentNavigator, String defaultStoreName,
+      BottomNavigationAnalytics bottomNavigationAnalytics, SearchAnalytics searchAnalytics) {
+    this.defaultStoreName = defaultStoreName;
+    this.bottomNavigationAnalytics = bottomNavigationAnalytics;
+    this.searchAnalytics = searchAnalytics;
+    bottomNavigationItens = new ArrayList<>();
+    this.fragmentNavigator = fragmentNavigator;
+  }
+
+  public void navigateToBottomNavigationItem(int bottomNavigationPosition) {
+    switch (bottomNavigationPosition) {
+      case 0:
```

Comment:
These attributes related to the position could be extracted to a field and reused on each of the navigateTo methods.
This way, it will be easier to change the "positions" in the future, without having to change it on more than one place.